bridge/ffi/src/bindings.rs

```diff
@@ -0,0 +1,203 @@
+use dsnp_graph_config::{ConnectionType, DsnpVersion, GraphKeyType, SchemaId};
+use dsnp_graph_core::dsnp::{
+	api_types::{Connection, PageHash, PageId},
+	dsnp_types::DsnpUserId,
+};
+use libc::{c_void, size_t};
+/// KeyData wrapper
+#[repr(C)]
+pub struct KeyData {
```

Comment:
If the Rust type is supported in C representation. Should we just use that `#[repr(C)]` instead of redefining all? What are the pros and cons of each choice?

Reply:
Vec is not supported, so we would need them as far as i found so far, but will clean if possible

Reply:
We can also use feature flag but that will make rust code look ugly